bootstrap/vars/addons.sample.yaml

```diff
@@ -42,3 +42,10 @@ system_upgrade_controller:
   # ansible configuration files. Running ansible against an already provisioned
   # cluster with this enabled might cause your cluster to be downgraded.
   enabled: false
+
+# https://github.com/morphy2k/rss-forwarder
+rss_fowarder:
```

Comment:
Would it be possible to rename this to `discord_template_notifier` and update the refs and ks/hr name? Also instead of `discord_webhook_url` it could just be `webhook_url`

Reply:
Applied changes